### PR TITLE
fix(call): add possibility to call user via context menu

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
add posibility to call user via context menu

**Which issue(s) this PR fixes** 🔨
[AP-1097](https://satellite-im.atlassian.net/browse/AP-1097)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/views/user/User.vue [hunk 1]
```diff
@@ -156,8 +161,23 @@ export default Vue.extend({
     this.$store.commit('ui/toggleContextMenu', false)
   },
   methods: {
-    testFunc() {
-      this.$Logger.log('User Context', 'Test func')
+    // testFunc() {
+    //   this.$Logger.log('User Context', 'Test func')
+    // },
```
Comment (Jekrimo) [inline components/views/user/User.vue:166]: Can we delete this bit?

File: components/views/user/User.vue [hunk 2]
```diff
@@ -65,14 +66,15 @@ export default Vue.extend({
...
     }),
     ...mapGetters('textile', ['getConversation']),
     contextMenuValues(): ContextMenuItem[] {
       return this.user.state === 'online'
         ? [
             { text: this.$t('context.send'), func: this.navigateToUser },
-            { text: this.$t('context.voice'), func: this.testFunc },
-            { text: this.$t('context.video'), func: this.testFunc },
+            { text: this.$t('context.voice'), func: this.handleCall },
+            // { text: this.$t('context.video'), func: this.testFunc },
```
Comment (Jekrimo) [inline components/views/user/User.vue:77]: Prob delete this too right?

**Review by Jekrimo — CHANGES_REQUESTED** [review]

File: components/views/user/User.vue (hunk 1 above)
Comment (josephmcg) [reply components/views/user/User.vue:166]: agreed

File: components/views/user/User.vue (hunk 2 above)
Comment (josephmcg) [reply components/views/user/User.vue:77]: this should stay, we're going to add it back eventually

File: components/views/user/User.vue [hunk 3]
```diff
@@ -65,14 +66,15 @@ export default Vue.extend({
       },
       textilePubkey: (state) =>
         (state as RootState).accounts?.details?.textilePubkey ?? '',
+      activeCall: (state) => (state as RootState).webrtc.activeCall,
     }),
     ...mapGetters('textile', ['getConversation']),
     contextMenuValues(): ContextMenuItem[] {
       return this.user.state === 'online'
```
Comment (josephmcg) [inline components/views/user/User.vue:73]: can use your new computed here

Comment (AlexZakablukov) [reply components/views/user/User.vue]: done

**Review by WanderingHogan — APPROVED** [review]

**Review by josephmcg — APPROVED** [review]

**Review by phillsatellite — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**